docs/content/direct/combined-status.md

```diff
@@ -0,0 +1,148 @@
+# Combined Status from WECs
+
+Note on terminology: the general idea that we wish to address is returning reported state about a workload object to its WDS. At the current level of development, we equate reported state with the status section of an object --- while anticipating a more general treatment in the future.
+
+There are two methods of returning reported state: a general one and a special case. The general method returns reported state from any number of WECs. The special case applies when the number of WECs is exactly 1, and returns the reported state into the original object in the WDS.
+
+## Introduction to the General Technique
+
+The general technique involves the following ideas.
+
+1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a small subset of the expression language.
+
+1. The specification of how to combine reported state appears in the BindingPolicy objects right next to the criteria for selecting workload objects. This saves users the trouble of having to write selection criteria twice.
+
+1. The combined reported state appears in a new kind of object, one per (workload object, BindingPolicy object) pair.
+
+1. A user can request a list without aggregation, possibly after filtering, but certainly with a limit on list length. The expectation is that such a list makes sense only if the length of the list will be modest. For users that want access to the full reported state from each WEC for a large number of WECs, KubeStellar should have an abstraction that gives the users access --- in a functional way, not by making another copy --- to that state (which is already in the mailbox namespaces).
```

Comment:
Is the mailbox still a concept now that we replaced IMBS with ITS?

Reply:
We still have mailbox namespaces in the ITS. Changing the term from IMBS to ITS does not change that. Also, it does not change the fact that the mailbox namespaces are an internal implementation detail.